net: lib: download_client: handle coap urls with multiple path elems (#6788)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Fixed discovery of empty services.

Libraries for networking
------------------------

* :ref:`lib_download_client` library:

  * Fixed

>     * An issue that would result in downloads of COAP URIs failing when they contained multiple path elements

Reviewer: ```suggestion
    * An issue where downloads of COAP URIs would fail when they contained multiple path elements.
```
